lib/native.js

```diff
@@ -27,7 +32,8 @@ native._dns01 = function (me, ch) {
 
 native._http01 = function (me, ch) {
 	return new me.request({
-		url: ch.challengeUrl
+		url: ch.challengeUrl,
+		agent: http01Agent
```

Comment:
Monkey patching `native._http01` in cases where an insecure agent is required would also be acceptable.